### DISCUSSION TITLE
Duration of Scenario is 0 after one step fail [messages]

### DISCUSSION THREAD

**Discussion by golinskiSt** [badeball/cypress-cucumber-preprocessor#961] [Q&A]
I want to ask if there was an issue or if this is known, but after test, generated messages shows that duration of scenario was equal to 0, but in Cypress i had 120 seconds of timeout. Why only Passed scenarios have provided time of execution?
![image](https://user-images.githubusercontent.com/30158798/223713055-b2230924-cc44-4182-9eca-0ed168bf50d7.png)

**Comment by badeball** [comment]
This has not been reported, feel free to open up a ticket.
